config/checker/com.intellij.astub

```diff
@@ -183,6 +210,20 @@ package com.intellij.openapi.project;
 @UIPackage package com.intellij.openapi.roots.ui.configuration.libraryEditor;
 @UIPackage package com.intellij.openapi.roots.ui.util;
 @UIPackage package com.intellij.openapi.ui;
+
+class DialogWrapper {
```

Comment:
I don't think that's a legitimate use for Checker stubs. Are these methods in `DialogWrapper` ever called from `DialogWrapper`'s or a subclass's c'tor? If not, then they're rather expecting `this` to be fully initialized.

Reply:
See comment to GitPushDialog.java, that's most likely not even needed.